client/components/single-pothole/SinglePotholeView.js

```diff
@@ -7,29 +7,60 @@ import {
   Dimmer,
   Loader,
   Image,
+  Dropdown,
 } from 'semantic-ui-react';
-import ReactMapboxGl, { Layer, Feature } from 'react-mapbox-gl';
+import GoogleMapReact from 'google-map-react';
 import mapAccessToken from '../../../secrets';
-import { createGotPotholeThunk } from '../../store';
+import { createGotPotholeThunk, createUpdateStatusThunk } from '../../store';
 import { connect } from 'react-redux';
 
-const Map = new ReactMapboxGl({
-  accessToken: mapAccessToken,
-});
-
-const style = 'mapbox://styles/mapbox/streets-v9';
+const accessToken = mapAccessToken;
```

Comment:
When we deploy this to Heroku we won't be using a secrets file. Instead, everything will be mapped to environment variables. Please create a process.env variable in the secrets file and reference it here. You shouldn't have to import it from the file explicitly